packages/discv5/test/unit/session/crypto.test.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-env mocha */
 import { expect } from "chai";
-import secp256k1 from "bcrypto/lib/secp256k1.js";
-import { randomBytes } from "bcrypto/lib/random.js";
+import * as secp256k1 from "@noble/secp256k1";
```

Comment:
Small thing, I think this file should be using getDiscv5Crypto instead of noble directly.